spring-cloud-alibaba-examples/integrated-example/integrated-account/pom.xml

```diff
@@ -30,9 +29,8 @@
 			<artifactId>spring-boot-starter-jdbc</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>mysql</groupId>
-			<artifactId>mysql-connector-java</artifactId>
-			<version>${mysql.version}</version>
+			<groupId>com.mysql</groupId>
+			<artifactId>mysql-connector-j</artifactId>
```

Comment:
Should it be `mysql-connector-java`?

Reply:
MySQL dependency has been migrated to this dependency

Reply:
Oh, I have checked that the artifact has been moved from mysql-connector-java to mysql-connector-j